src/utils/init/config-github.js

```diff
@@ -193,7 +217,7 @@ module.exports = async function configGithub({ context, repoName, repoOwner, sit
   })
   await saveNetlifyToml({ repositoryRoot, config, configPath, baseDir, buildCmd, buildDir, functionsDir })
 
-  const octokit = getGitHubClient({ token })
+  const octokit = await getGitHubClient(token)
```

Comment:
Is this `await` needed (same for the tests)?

Reply:
oh this is a old left over will change

Reply:
What do you mean with the same for the tests? In the tests a different function is tested `getGitHubToken` this is actually async

Reply:
Ah, I missed that part (the fact it's async in the tests). You can ignore that part of my comment